test/browser/FOAMByExampleRunner.js

```diff
@@ -191,4 +191,14 @@ foam.async.repeat(FBE.length, function runExemplar(index) {
     writeOutput();
     return result;
   }
-})();
+})().then(function() {
+  // reverting logging strategy performed during async calls
+  console = {
+    __proto__: oldConsole,
```

Comment:
This looks right, although we don't need to override `oldConsole`'s implementations with `oldConsole`'s implementations. `console = oldConsole` should suffice.